ledger-service/utils/src/main/scala/com/digitalasset/ledger/service/LedgerReader.scala

```diff
@@ -3,12 +3,15 @@
 
 package com.daml.ledger.service
 
+import com.daml.ledger.api.domain.LedgerId
 import com.daml.lf.archive
 import com.daml.lf.data.Ref.{Identifier, PackageId}
 import com.daml.lf.iface.reader.InterfaceReader
 import com.daml.lf.iface.{DefDataType, Interface}
 import com.daml.ledger.api.v1.package_service.GetPackageResponse
 import com.daml.ledger.client.services.pkg.PackageClient
+import com.daml.ledger.client.services.pkg.withoutledgerid.{PackageClient => LoosePackageClient}
+//import com.daml.ledger.rxjava.PackageClient
```

Comment:
No need to comment out code, just drop it.

Reply:
good catch, thanks